src/globus_cli/commands/streams/tunnel/delete.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command
+from globus_cli.termio import display
+
+
+@command("delete", short_help="Delete a Globus Tunnel.")
```

Comment:
Tunnel (as a name for the resource) appears in uppercase and lowercase in this PR. Recommending lowercase for consistency.

```suggestion
@command("delete", short_help="Delete a Globus tunnel.")
```